agora/Agora/Proposal/Time.hs

```diff
@@ -103,6 +103,7 @@ data ProposalTimingConfig = ProposalTimingConfig
   -- ^ "L": the length of the locking period.
   , executingTime :: POSIXTime
   -- ^ "E": the length of the execution period.
+  , unlockCooldown :: POSIXTime
```

Comment:
Regarding naming, a lock is generic to multiple actions (create, cosign, vote). This cooldown is specific to just the voting one. What do you think about something like `minStakeVotingTime` instead?

Reply:
Hmm, good idea.